### PR TITLE
Package asak.0.4

### PR DESCRIPTION
**Pull request by nobrakal**
### `asak.0.4`
Partitioning OCaml codes
Asak provides functions to parse, type-check and identify similar OCaml codes.
These functions are then used to partition codes implementing the same function and help to detect code duplication.

---
* Homepage: https://github.com/nobrakal/asak
* Source repo: git+https://github.com/nobrakal/asak.git
* Bug tracker: https://github.com/nobrakal/asak/issues

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

File: packages/asak/asak.0.4/opam [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+license: "MIT"
+
+homepage: "https://github.com/nobrakal/asak"
+doc: "https://nobrakal.github.io/asak/asak/index.html"
+bug-reports: "https://github.com/nobrakal/asak/issues"
+dev-repo: "git+https://github.com/nobrakal/asak.git"
+
+depends: [
+  "ocaml"    {>= "4.05" & < "4.15"}
+  "cmdliner" {>= "1.0.0"}
```
Comment (mseri) [inline packages/asak/asak.0.4/opam:21]:
```suggestion
  "cmdliner" {>= "1.1.0"}
```
For 

    # File "bin/anzad.ml", line 199, characters 2-10:
    # Error: Unbound module Cmd

File: packages/asak/asak.0.4/opam [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+authors: ["Alexandre Moine"]
+license: "MIT"
+
+homepage: "https://github.com/nobrakal/asak"
+doc: "https://nobrakal.github.io/asak/asak/index.html"
+bug-reports: "https://github.com/nobrakal/asak/issues"
+dev-repo: "git+https://github.com/nobrakal/asak.git"
+
+depends: [
+  "ocaml"    {>= "4.05" & < "4.15"}
```
Comment (mseri) [inline packages/asak/asak.0.4/opam:20]:
```suggestion
  "ocaml"    {>= "4.10" & < "4.15"}
```
Due to
    # File "src/partition.ml", line 54, characters 14-22:
    # Error: Unbound value find_map

**Comment by nobrakal** [comment]
Thanks for the review. I see that you applied the changes you suggested, is there anything that I should do now?

**Comment by mseri** [comment]
Port those fixes upstream please (without retagging or rereleasing). If the build passes it should be good to merge

**Comment by nobrakal** [comment]
Thanks for your help. 
Opam's CI caught a bug on older OCaml versions that I now fixed.
What is the procedure? Cancel this PR, rettag after the fix, and make another fresh PR?

**Comment by mseri** [comment]
You can also force push on top of this one

**Comment by AltGr** [comment]
Thanks a lot @nobrakal

**Comment by mseri** [comment]
All green now!

File: packages/asak/asak.0.4/opam [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+authors: ["Alexandre Moine"]
+license: "MIT"
+
+homepage: "https://github.com/nobrakal/asak"
+doc: "https://nobrakal.github.io/asak/asak/index.html"
+bug-reports: "https://github.com/nobrakal/asak/issues"
+dev-repo: "git+https://github.com/nobrakal/asak.git"
+
+depends: [
+  "ocaml"    {>= "4.10" & < "4.15"}
```
Comment (avsm) [inline packages/asak/asak.0.4/opam:20]:
```suggestion
  "ocaml"    {>= "4.10" & < "4.15~~"}
```

Comment (avsm) [reply packages/asak/asak.0.4/opam:20]:
```suggestion
  "ocaml"    {>= "4.10" & < "4.15~~"}
```

**Comment by mseri** [comment]
Good point. Why the double tilde?

**Pull request merged**